Source/WebCore/dom/EventSender.h

```diff
@@ -121,7 +121,7 @@ template<typename T, typename WeakPtrImpl> void EventSender<T, WeakPtrImpl>::dis
         if (!weakSender)
             continue;
         auto sender = std::exchange(weakSender, nullptr);
-        if (!page || sender->document().page() == page)
```

Comment:
page() is definitely a trivial getter so we shouldn't have to use protectedDocument here.

Reply:
Yes, not sure what I was thinking :) Fixed.